composer.json

```diff
@@ -42,7 +41,6 @@
         "drupal/prlp": "^1.3",
         "drupal/redirect": "^1.3",
         "drupal/role_delegation": "^1.0-alpha1",
-        "drupal/scheduled_updates": "^1.0-alpha7",
```

Comment:
@vincent-gao do we need an update hook to disable the module? Have we tried this on a clean install and checked that it doesn't complain about module missing from file system?

Reply:
yes we need an update hook to uninstall the module. Make sure that the module is removed from all dependencies.

Reply:
yes. updated an update_hook for this purpose